guides/release/contributing/repositories.md

```diff
@@ -17,10 +17,12 @@ Ember is made up of several libraries. If you wish to add a feature or fix a bug
 
 * [https://github.com/emberjs/website](https://github.com/emberjs/website)
 
-**Ember Guides** - Source for [http://guides.emberjs.com](http://guides.emberjs.com) which you are currently reading.
+**Ember Deprecations** - Source for [https://deprecations.emberjs.com] (https://deprecations.emberjs.com)
+* [https://github.com/ember-learn/deprecation-app](https://github.com/ember-learn/deprecation-app)
```

Comment:
Based on the description called out in the issue, I assumed that the deprecations guide should have its own section.  I am not sure about placement on the screen so I arbitrarily picked where it is here.

Let me know if you have a location on the page that would make more sense to you.